README.md

```diff
@@ -31,7 +31,7 @@ Please see the [contribution guidelines](CONTRIBUTING.md).
 - Processes
   - Developer Operations
   - [Quality Assurance](docs/processes/quality-assurance/quality-assurance-overview.md)
-  - [Documentation](docs/documentation/index.md)
+  - [Documentation](docs/processes/documentation/documentation.md)
```

Comment:
**issue:** Is this change intended? We already had the file `docs/documentation/index.md` in the `main` branch. This change will effectively remove its reference from this file.

Reply:
As per above, I was going to merge the files into one once merged and keep the index name for consistency. But did not want to 'rob' Shivam of the commit evidence

Reply:
> But did not want to 'rob' Shivam of the commit evidence

Gotcha. It is not really robing by the way. Shivam's change is still presented in this PR. We don't want to merge changes that are conflicting or duplicating.

By the way, I have removed the prettier and package changes in this PR. Please proceed with updating the documentation references and content whenever you see fit.